src/Components/Footer/Footer.jsx

```diff
@@ -33,9 +34,9 @@ function Footer() {
 						<h2>Contact</h2>
 						<div className="footer_l__hr_text"><hr color="#29abe2" /></div><br /><br /><br /></div>
 				</div>
-				<div className="footer_l__contact_content"><a className="footer_l__contact_cn" >Email: cerebro@iiitv.ac.in                             </a></div>
-				<div className="footer_l__contact_content"><a className="footer_l__contact_cn" >Phone: +91-9370645038</a></div>
-				<div className="footer_l__contact_content"><a className="footer_l__contact_cn" >Address: c/o Block No.9, Government Engineering<div className="footer_l__contact_gmap">College, Sector-28, Gandhinagar, Gujarat - 382028</div></a></div>
+					<div className="footer_l__contact_content"><a href='mailto:cerebro@iiitvadodara.ac.in' className="footer_l__contact_cn" ><b>Email</b>: cerebro@iiitvadodara.ac.in                             </a></div>
```

Comment:
Add ```target="_blank"``` here too because many users use browsers instead of mail clients to open mail links.